Kickstarter-iOS/Views/Cells/ProjectPamphletMainCell.swift

```diff
@@ -248,6 +279,8 @@ internal final class ProjectPamphletMainCell: UITableViewCell, ValueCell {
       |> UIStackView.lens.isAccessibilityElement .~ true
       |> UIStackView.lens.backgroundColor .~ .white
 
+    _ = self.youreABackerContainerViewLeadingConstraint.constant = leftRightInsetValue
```

Comment:
What's this unused argument on the left? 🤔

Reply:
Ah, apologies! A poor copy-paste job!